data-raw/ictr_data.R

```diff
@@ -1,12 +1,14 @@
+library(vroom)
 library(here)
-library(readr)
 library(usethis)
+library(dplyr)
+devtools::load_all()
```

Comment:
FYI I think we need dplyr to use `select()` and `devtools::load_all()` to use `ictr_inputs_crucial()`, right?